src/resources/postcss/tribe-events-v2-block-editor-reskin.pcss

```diff
@@ -0,0 +1,42 @@
+/**
+ * Block editor event single v2 reskin
+ *
+ * @since TBD
+ */
+
+/* = Website block view styles
+=============================================*/
+.tribe-block__event-website {
+    padding: 20px 0;
+
+    a {
+        border: none;
+        background-color: var(--tec-color-link-accent);
```

Comment:
do we want this to use variables (so customizer can take effect)? if so, we should set variables as a dependency of this file. also use variables for all things below (font-weight, padding, color, etc.)

Reply:
Currently digging into this, I'll make an update asap.

Reply:
I've added variables here as well